ccd-definition/FixedLists.json

```diff
@@ -513,5 +513,89 @@
     "ID": "PartyType",
     "ListElementCode": "ORGANISATION",
     "ListElement": "Organisation"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeList",
+    "ListElementCode": "CASE_MANAGEMENT",
+    "ListElement": "Case management",
+    "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeList",
+    "ListElementCode": "FURTHER_CASE_MANAGEMENT",
+    "ListElement": "Further case management",
+    "DisplayOrder": 2
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeList",
+    "ListElementCode": "ISSUE_RESOLUTION",
+    "ListElement": "Issue resolution",
+    "DisplayOrder": 3
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeList",
+    "ListElementCode": "FINAL",
+    "ListElement": "Final",
+    "DisplayOrder": 4
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingTypeList",
+    "ListElementCode": "OTHER",
+    "ListElement": "Other",
+    "DisplayOrder": 5
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "HearingNeedsBookedList",
```

Comment:
<img width="567" alt="Screenshot 2019-08-29 at 08 30 35" src="https://user-images.githubusercontent.com/4429628/63919754-4a4bbb80-ca37-11e9-833e-d2e8e09f4f12.png">

I don't like `hearingNeedsBooked`. Even in the prototype I don't fully understand what the question is asking me. Could we question this?

Reply:
👍